components-compose/src/main/java/uk/gov/hmrc/components/compose/ui/theme/HmrcTypography.kt

```diff
@@ -67,5 +67,11 @@ data class HmrcTypography(
         fontWeight = FontWeight.Normal,
         fontSize = 16.sp,
         color = hmrcRed
+    ),
+    val bulletedText: TextStyle = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Normal,
+        fontSize = 16.sp,
+        color = hmrcBlack
     )
```

Comment:
Do we need to define this as it's own typography item? It's the same as the `body` text and we would always want it to be consistent with the body text I think? If we used the `bulletedText` as defined here in the code somewhere, it wouldn't come with a bullet so may be a little misleading? I think we can just use `body` text in the bullet component.